bluemira/base/parameter_frame/_frame.py

```diff
@@ -96,3 +96,33 @@ def from_json(cls, json_in: Union[str, json.SupportsRead]) -> NewParameterFrame:
                 return cls.from_dict(json.load(f))
         # load from a JSON string
         return cls.from_dict(json.loads(json_in))
+
+
+def parameter_setup(
```

Comment:
I still feel this should be `make_parameter_frame` (or similar). It's pretty common throughout Bluemira to use `make_` for factory functions, and `parameter` is different to `parameter_frame`

Reply:
done